lib/msf/core/post/windows/wmic.rb

```diff
@@ -47,7 +45,7 @@ def wmic_query(query, server=datastore['RHOST'])
       end
     end
 
-    if extapi && !is_system?
+    if !is_system? && session.commands.include?(Rex::Post::Meterpreter::Extensions::Extapi::COMMAND_ID_EXTAPI_CLIPBOARD_SET_DATA)
```

Comment:
```suggestion
    if session.commands.include?(Rex::Post::Meterpreter::Extensions::Extapi::COMMAND_ID_EXTAPI_CLIPBOARD_SET_DATA) && !is_system? 
```
Think we should probably keep the order the same here and for the other one below

Reply:
y

Reply:
changed